WebClient/src/site.ts

```diff
@@ -0,0 +1,44 @@
+﻿import * as signalR from "@microsoft/signalr";
+import { Communicator } from "./communicator";
+
+let comm = new Communicator();
+comm.startConnection();
+console.log(comm.connection);//test
+
+
+comm.connection.on("ReceiveMessage", function (user : string, message : string) {
```

Comment:
The "ReceiveMessage" and "ReceiveGroup" subscribes should be  the communicator's task, so this should be moved inside.